code/src/css/movieDetails.css

```diff
@@ -0,0 +1,103 @@
+.details-page {
+  min-height: 85vh;
+  padding: 30px;
+  background-size: cover;
+}
+
+.back-button {
+  font-weight: bold;
+}
+
+.fas {
+  transform: scale(1.3);
+  margin-right: 7px;
+  transition: margin 300ms ease;
+}
+
+.back-button:hover .fas {
+  margin-right: 12px;
+}
+
+.movie-details-container {
+  min-height: 80vh;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  flex-wrap: wrap;
+  padding-top: 30px;
+}
+
+.movie-details-image {
+  border: 5px solid #fff;
+  width: 100%;
+  max-width: 342px;
+  margin-bottom: 10px;
+}
+
+.movie-details {
+  max-width: 310px;
+  margin-top: 15px;
+  background: rgba(0, 0, 0, 0.5);
+  padding: 30px 20px;
+  border-radius: 30px 3px;
+}
+
+.movie-details:hover {
+  background: rgba(0, 0, 0, 0.8);
+}
```

Comment:
I really liked the nice background you have behind the overview-text. It makes it easy to read the next with all possible backgrounds.